### PR TITLE
PLT-4287 Use a subscription to get access to functionalities in your tier

### PR DESCRIPTION
**Pull request by amnambiar**
https://input-output.atlassian.net/browse/PLT-4287

### CODE REVIEW

File: react-web/src/pages/certification/Certification.tsx [hunk 1]
```diff
@@ -248,7 +249,11 @@ const Certification = () => {
 
   return (
     <>
-      <div
+      {!subscribedFeatures?.length || subscribedFeatures?.indexOf("l1-run") === -1 ?
```
Comment (bogdan-manole) [inline react-web/src/pages/certification/Certification.tsx:252]: this should be enough 
```suggestion
      {subscribedFeatures?.indexOf("l1-run") === -1 ? 
```

File: react-web/src/pages/certification/Certification.tsx [hunk 2]
```diff
@@ -248,7 +249,11 @@ const Certification = () => {
 
   return (
     <>
-      <div
+      {!subscribedFeatures?.length || subscribedFeatures?.indexOf("l1-run") === -1 ? 
+      <div id="searchContainerEmpty">
+        <p>You do not have enough subscriptions to perform the test run. <br/> <Link to="/subscription">Click here</Link> to review our packages and pick one to proceed.</p>
```
Comment (bogdan-manole) [inline react-web/src/pages/certification/Certification.tsx:254]: my English is far from being my strong point, but this sounds a bit strange to me. 
How many subscriptions do I need to perform a test? :) 
Shouldn't be more related to something like: you don't have a valid subscription to perform bla bla bla?

**Review by bogdan-manole — CHANGES_REQUESTED** [review]
looks good to me 👍 , just some small comments

File: react-web/src/pages/certification/Certification.tsx (hunk 1 above)
Comment (amnambiar) [reply react-web/src/pages/certification/Certification.tsx]: Done

File: react-web/src/pages/certification/Certification.tsx (hunk 2 above)
Comment (amnambiar) [reply react-web/src/pages/certification/Certification.tsx]: Done

**Comment by bogdan-manole** [comment]
now seems right, after solving commits, it will be fit for an approval

**Review by bogdan-manole — APPROVED** [review]

**Review by RSoulatIOHK — APPROVED** [review]

**Pull request merged**